recipes/Semi-ATE/LICENSE.txt

```diff
@@ -0,0 +1,280 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Since the PyPI tarball already includes `LICENSE.txt` you don't need to (and shouldn't) include it here.

Reply:
@dopplershift : I was asked in the past to also add the License even though it was already present on PyPI. Anyway, your request makes more sense as this eliminates possible License conflicts (copy wrong License by accident) :smile: